packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -231,6 +231,8 @@ export abstract class FluidDataStoreContext extends TypedEventEmitter<IFluidData
     protected registry: IFluidDataStoreRegistry | undefined;
 
     protected detachedRuntimeCreation = false;
+    // @ts-expect-error - This shouldn't be referenced in the current version, but needs to be here for back-compat
```

Comment:
can we mark it with "// back-compat" comment and mention when it can be removed?